prow/config.yaml

```diff
@@ -1266,6 +1266,33 @@ presubmits:
           requests:
             memory: "6Gi"
 
+  - name: pull-kubernetes-e2e-kubeadm-dind-ipv6
+    context: pull-kubernetes-e2e-kubeadm-dind-ipv6
+    max_concurrency: 8
+    skip_report: true
```

Comment:
please also set `always_run: false`, new / unproven jobs should run by manual request